### PR TITLE
Write outputs in text files (fixes #52)

### LINKED ISSUE #52: Write outputs in text files
* Modify option `-d` to pass only a basename for the output file names. Use this basename to write the following files (assuming base name is `concordia`):
  1. `concordia_differences_subject_total.txt`: currently written at lines 560-. Change the output file name.
  2. `concordia_differences.txt`: currently written at line 567. Change the output file name. Remove the `args.binaryMatrix` argument.
  3. `concordia_row_index.txt`
  4. `concordia_column_index.txt`
* Document this in the `-h` help. 
* `-d` should become a mandatory parameter.
* Remove matrix printings on standard output (line 570).

### PR DESCRIPTION
**Pull request by soudabeh19**
Fixes #52

### CODE REVIEW

File: verifyFiles.py [hunk 1]
```diff
@@ -554,7 +554,7 @@ def main():
         #diff,metric_values,dictionary_executables,dictionary_processes=n_differences_across_subjects(conditions_dict,root_dir,metrics,checksums_from_file_dict,args.checksumFile,args.checkCorruption,args.sqLiteFile)i
 	diff,bDiff,metric_values,dictionary_executables,dictionary_processes,metric_values_subject_wise=n_differences_across_subjects(conditions_dict,root_dir,metrics,checksums_from_file_dict,args.checksumFile,args.checkCorruption,args.sqLiteFile,args.trackProcesses)
        	if args.difference is not None:
-            log_info("Writes the difference matrices and indexs into files"+args.difference)
+            log_info("Writes the difference matrices and indexs into files")
```
Comment (lalet) [inline verifyFiles.py]: Typo : Indexs = > Indexes

**Review by lalet — CHANGES_REQUESTED** [review]

File: verifyFiles.py [hunk 2]
```diff
@@ -512,7 +512,7 @@ def main():
                                              /home/$(USER)/CentOS7.FSL5.0.6
                                              Each directory will contain subject folders like 100307,100308 etc'''))
         parser.add_argument("-c", "--checksumFile",help="Reads checksum from files. Doesn't compute checksums locally")
-	parser.add_argument("difference", help="Writes the difference matrices and indexs into files")
+	parser.add_argument("difference", help="Base name to use in output file names")
```
Comment (glatard) [inline verifyFiles.py]: Rename parameter `result_base_name`. Explain in the help that the following files will be written: <result_base_name>_....txt (small explanation), etc.

**Review by glatard — CHANGES_REQUESTED** [review]
Just improve the help message and rename the parameters.

**Comment by lalet** [comment]
@soudabeh19 I guess you haven't made a pull from upstream.

**Pull request merged**